packages/runtime-core/src/customFormatter.ts

```diff
@@ -34,13 +36,16 @@ export function initCustomFormatter(): void {
       if (obj.__isVue) {
         return ['div', vueStyle, `VueInstance`]
       } else if (isRef(obj)) {
+        // avoid tracking during debugger accessing
+        pauseTracking()
+        const value = obj.value
+        resetTracking()
         return [
           'div',
           {},
           ['span', vueStyle, genRefFlag(obj)],
           '<',
-          // avoid debugger accessing value affecting behavior
```

Comment:
This makes me wonder if it wasn't intentional to completely avoid calling the getter

Reply:
It was originally `obj.value`, but it was changed to the current implementation in https://github.com/vuejs/core/pull/10397/files#diff-95f4e5cf8305f906512f733429443451392894309ce012c5a70216f9e578738eL41. I think this change may not have considered the case where the value of `._value` is `undefined` when `.value` is not accessed. So I think we should continue using `obj.value`, but not track it.

![image](https://github.com/user-attachments/assets/8c4c2aa4-e6d2-4c1b-867c-8ade2502fb09)

Reply:
Sounds good to me. I thought that maybe displaying `undefined` for a value that was never accessed before was the expected behavior